plugins/couchbase_logs.yaml

```diff
@@ -0,0 +1,253 @@
+            layout: '%Y-%m-%dT%H:%M:%S.%L%j'
+          severity:
+            parse_from: attributes.couchbase_severity
+            mapping:
+              warn: warn
+    {{ end }} ## end info log receiver
+
+    {{ if .enable_access_log}}
+    filelog/access_logs:
+      include: 
+        {{ range $fp := .http_access_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'couchbase.http.access'
+        plugin_id: 'Couchbase'
+      operators:
+        - type: regex_parser
+          regex: '(?P<ip_address>\d+.\d+.\d+.\d+)\s+(?P<user_identifier>.*) \[(?P<timestamp>\d{2}\/\w+\/\d{4}:\d{2}:\d{2}:\d{2} -\d{4})\] \\*"(?P<request_line>[^"]*)\\*" (?P<status>\d{3}) (?P<size>\d*) *-*(?P<message>.*)'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%d/%b/%Y:%H:%M:%S %z'
+          severity:
+            parse_from: attributes.status
+            preset: none
+            mapping:
+              info: 2xx
+              info2: 3xx
+              warn: 4xx
+              error: 5xx
+    {{ end }} ## end http access log receiver
+
+    {{ if .enable_internal_access_log}}
+    filelog/internal_access_logs:
+      include: 
+        {{ range $fp := .http_internal_access_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'couchbase.http.access.internal'
+        plugin_id: 'Couchbase'
+      operators:
+        - type: regex_parser
+          regex: '(?P<ip_address>\d+.\d+.\d+.\d+)\s+(?P<user_identifier>.*) \[(?P<timestamp>\d{2}\/\w+\/\d{4}:\d{2}:\d{2}:\d{2} -\d{4})\] \\*"(?P<request_line>[^"]*)\\*" (?P<status>\d{3}) (?P<size>\d*) *-*(?P<message>.*)'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%d/%b/%Y:%H:%M:%S %z'
+          severity:
+            parse_from: attributes.status
+            preset: none
+            mapping:
+              info: 2xx
+              info2: 3xx
+              warn: 4xx
+              error: 5xx
+    {{ end }} ## end internal access log receiver
+
+
+    {{ if .enable_babysitter_log}}
+    filelog/babysitter_logs:
+      include: 
+        {{ range $fp := .babysitter_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline: 
+        line_start_pattern: '\[[a-zA-Z_]+:'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'couchbase.babysitter'
+        plugin_id: 'Couchbase'
+      operators:
+        - type: regex_parser
+          regex: '^\[(?P<type>[^:]*):(?P<couchbase_severity>[^,]*),(?P<timestamp>\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d+-\d{2}:\d{2}),(?P<node>[^@]*)@(?P<host>[^:]*):(?P<source>[^\]]+)\](?P<message>(?s).*[^\n\s])[\n\s]*$'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%Y-%m-%dT%H:%M:%S.%L%j'
+          severity:
+            parse_from: attributes.couchbase_severity
```

Comment:
Also wondering if these logs have "WARNING" as their severity as opposed to "WARN"